### ISSUE TITLE
md-input placeholder issue with binding / translate

### ISSUE THREAD

**Issue by chrismbeckett** [angular/material#3196]
I am using an md-input with an icon, and floating label configuration. This matches the sample code found on the website.

```html
<md-input-container>
  <md-icon md-font-library="material-icons">email</md-icon>
  <input
         type="email"
         ng-required="true"
         name="email"
         placeholder="{{ctrl.placeholders.email}}"
         ng-model="ctrl.input.email">
  <div ng-if="ctrl.form.$submitted" ng-messages="ctrl.form.email.$error">
    <div ng-message="required">An email address is required</div>
    <div ng-message="email">The email address is invalid</div>
  </div>
</md-input-container>
```

If I try to use any sort of expression for the placeholder rather than a hard-coded string, both a placeholder a label are rendered, instead of just a placeholder. In the example below, I use the same markup for both email and password. For email, I bind the placeholder, and for password I have a hard-coded string.

![input_bug1](https://cloud.githubusercontent.com/assets/6036093/8076020/3aca5f7c-0efc-11e5-9370-2f6996376a29.png)
![input_bug2](https://cloud.githubusercontent.com/assets/6036093/8076021/3c69f31a-0efc-11e5-9d35-81ed67574762.png)

NOTE: I also noticed that the aria-label is set to the un-processed expression "{{ctrl.placeholders.email}}" rather than the evaluated string.

**Comment by Splaktar** [comment]
This should be fixed in 0.10.0. Can you confirm that please? Also in what browser were you seeing this?

**Comment by chrismbeckett** [comment]
No, it's not fixed in 010.0. I thought I was using 0.10.0, but was on 0.9.8. Upgrading managing to break a few other things in my app, but this is still broken.

- Angular 1.4.1
- Angular Material 0.10.0
- reproduced with identical behavior in IE11, Chrome 43, FIrefox Developer 40, Chrome 43 on Android 5, and Mobile Safari on iOS 8.4.

If you can ping me on Gitter, I can provide a private URL that can demonstrate the problem.

**Comment by chrismbeckett** [comment]
Fixed.

**Comment by Splaktar** [comment]
Yes, 'normal placeholders' should be fixed with #3582. Thanks for checking on this.

**Comment by Burtan** [comment]
I have this problem with 1.1.0-rc2 and 1.0.7 using translated or expressioned floating labels/placeholders with chrome 50 and firefox 45.1.1
